### ISSUE TITLE
This is not really an issue, but rather comment

### ISSUE THREAD

**Issue by Sadrien** [frifox/gw2dps#50]
This automatically runs and functions on gw2-64.exe correct? I see the gw2dps-64.dlls and assume this is true.

**Comment by Sadrien** [comment]
Also, has anyone been banned while using this other than frifox?

**Comment by snakelayer** [comment]
I have not used this recently, but the last time I tried, it still functions with 64-bit gw2.

Frifox was banned from using this? that's news to me...

**Comment by frifox** [comment]
Correct, I was banned for using other 3rd party apps to fly and walk through walls. Using various tools to directly tamper with gw2 memory and posting videos of doing it online probably played a part in the termination too.

Using GW2DPS had nothing to do with termination. My other accounts on which I regularly used gw2dps (but never the "other" tools) are still active.
